Update databases.md (#40)
moj-analytical-services/user-guidance: source/documentation/data-docs/curated-databases-docs/dbtools.md

# dbtools

`dbtools` is a simple package, available for R and Python that allows you to query databases using Amazon Athena and bring them directly into your R and Python environment as a DataFrame.

## R

> Please visit the repository for the R package [dbtools](https://github.com/moj-analytical-services/dbtools/) which gives details as to how to install and use the package.

Reviewer: ```suggestion
The [README](https://github.com/moj-analytical-services/dbtools/blob/master/README.md) in the [dbtools](https://github.com/moj-analytical-services/dbtools/) repository gives details on how to install and use the package with R.
```

## Python

> Please visit the repository for the Python package [pydbtools](https://github.com/moj-analytical-services/pydbtools/) which gives details as to how to install and use the package.

Reviewer: ```suggestion
The [README](https://github.com/moj-analytical-services/pydbtools/blob/master/README.md) in the [pydbtools](https://github.com/moj-analytical-services/pydbtools/) repository gives details on how to install and use the package with Python.
```
